lib/Runtime/Library/RuntimeFunction.cpp

```diff
@@ -59,6 +59,11 @@ namespace Js
         this->functionNameId = nameId;
     }
 
+    bool RuntimeFunction::Is(Var func)
+    {
+        return JavascriptFunction::Is(func) && !JavascriptFunction::UnsafeFromVar(func)->GetFunctionInfo()->HasBody();
```

Comment:
@curtisman does this look right?

For reference, ScriptFunction is defined similarly (without negating HasBody()):

```
    bool ScriptFunction::Is(Var func)
    {
        return JavascriptFunction::Is(func) && JavascriptFunction::UnsafeFromVar(func)->GetFunctionInfo()->HasBody();
    }
```